portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -21,6 +21,8 @@
 import com.google.appengine.api.datastore.PreparedQuery;
```

Comment:
If you wanted to lessen the amount of import lines at the beginning, you could do import com.google.appengine.api.datastore.* like you did for other files, but there's no limit for import for import statements in the style guide.